deploy/scripts/src/ais/test.ts

```diff
@@ -0,0 +1,158 @@
+import { AWSConfig, SQSClient } from '../common/utils/jslib/aws-sqs'
+import { generatePersistIVRequest } from './requestGenerator/aisReqGen'
+import { type AssumeRoleOutput } from '../common/utils/aws/types'
+import { uuidv4 } from '../common/utils/jslib/index'
+import { group } from 'k6'
+import { timeRequest } from '../common/utils/request/timing'
+import { isStatusCode200, pageContentCheck } from '../common/utils/checks/assertions'
+import { SharedArray } from 'k6/data'
+import http from 'k6/http'
+
+const profiles: ProfileList = {
+  smoke: {
+    persistIV: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 5,
+      stages: [
+        { target: 1, duration: '60s' } // Ramps up to target load
+      ],
+      exec: 'persistIV'
+    },
+    retrieveIV: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 5,
+      stages: [
+        { target: 1, duration: '60s' } // Ramps up to target load
+      ],
+      exec: 'retrieveIV'
+    }
+  },
+  lowVolumeTest: {
+    persistIV: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 150,
+      stages: [
+        { target: 30, duration: '15m' }, // Ramp up to 30 iterations/messages per second in 15 minutes
+        { target: 30, duration: '30m' }, // Maintain a steady state of 30 messages per second for 30 minutes
+        { target: 0, duration: '5m' } // Total ramp down in 5 minutes
+      ],
+      exec: 'persistIV'
+    },
+    retrieveIV: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 150,
+      stages: [
+        { target: 100, duration: '15m' }, // Ramp up to 100 iterations per second in 15 minutes
+        { target: 100, duration: '30m' }, // Maintain a steady state of 100 iterations per second for 30 minutes
+        { target: 0, duration: '5m' } // Total ramp down in 5 minutes
+      ],
+      exec: 'retrieveIV'
+    }
+  },
+  stress: {
+    persistIV: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 500,
+      stages: [
+        { target: 100, duration: '15m' }, // Ramp up to 100 iterations/messages per second in 15 minutes
+        { target: 100, duration: '30m' }, // Maintain a steady state of 100 messages per second for 30 minutes
+        { target: 0, duration: '5m' } // Total ramp down in 5 minutes
+      ],
+      exec: 'persistIV'
+    },
+    retrieveIV: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 11400,
+      stages: [
+        { target: 3800, duration: '15m' }, // Ramp up to 3800 iterations per second in 15 minutes
```

Comment:
Should this be 1900 and two calls in the scenario?

Reply:
No the worst case scenario would be the peak volume of calls from Auth (1900/s) and Core (1900/s) simultaneously. The GET call from both Auth and Core is same so testing this at 3800/s would be the worst case peak volume of this GET call.